experimental_parameter_space/evaluate_permeation_number_W.py

```diff
@@ -0,0 +1,117 @@
+import numpy as np
+import festim as F
+import matplotlib.pyplot as plt
+from matplotlib.colors import TwoSlopeNorm
+
+from festim_sim import (
+    substrate_D_0,
+    substrate_E_D,
+    substrate_S_0,
+    substrate_E_S,
+    substrate_Kd_0,
+    substrate_E_Kd,
+)
+
+k_b = F.k_B
+
+
+def W_number(K_d, thickness, pressure, diffusivity, solubility):
+    return (K_d * (pressure**0.5) * thickness) / (diffusivity * solubility)
+
+
+default_P = 1e4  # Pa
+default_e = 974e-6  # m
+default_T = 300 + 273.15  # K
+
+
+def W_testing(P=default_P, e=default_e, T=default_T):
+
+    return W_number(
+        K_d=substrate_Kd_0 * np.exp(-substrate_E_Kd / (k_b * T)),
+        thickness=e,
+        pressure=P,
+        diffusivity=substrate_D_0 * np.exp(-substrate_E_D / (k_b * T)),
+        solubility=substrate_S_0 * np.exp(-substrate_E_S / (k_b * T)),
+    )
+
+
+P_testing = np.geomspace(
+    1e2, 1e05, num=100
+)  # Pa  range taken from https://doi.org/10.1016/j.nme.2021.101062 Section 2.5
+e_testing = np.geomspace(
+    2e-4, 1e-3, num=100
+)  # m minimum value taken from https://doi.org/10.1016/j.nme.2021.101062 Section 2.5
+T_testing = np.linspace(200, 400, num=100) + 273.15  # K
+
+e_ticks = np.linspace(2e-4, 1e-3, num=9)
+e_plot = [0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0]
+
+plt.rc("text", usetex=True)
+plt.rc("font", family="serif", size=12)
```

Comment:
Please remove this, it requires people to have latex installed to run this, otherwise need to comment it, utterly annoying and not needed at this stage! 😸

Reply:
will do